tests/chainerx_tests/unit_tests/routines_tests/test_rnn.py

```diff
@@ -429,6 +429,7 @@ def forward_chainer(self, inputs):
 
 
 @op_utils.op_test(['native:0', 'cuda:0'])
+@op_utils.fix_random()  # ReLU activation is unstable around 0 but can seemingly not be dodged automatically.  # NOQA
```

Comment:
Why `NOQA`? If it's because of a long line, please place a comment at the line above.

Reply:
Fixed as suggested.